### ISSUE TITLE
MenuEntrySwapper not working for some objects/items

### ISSUE THREAD

**Issue by SRLJustin** [runelite-extended/runelite#805]
Running latest build,
https://i.imgur.com/wKRgtTi.png

Plugin is enabled with the option ticked:
https://i.imgur.com/IDQOhEl.png

MenuEntrySwapperPlugin:
```
		else if (config.swapQuick() && option.equals("open"))
		{
			swap(client, "quick-open", option, target, true);
		}
```

Remove Objects not working in MenuEntrySwapper
https://i.imgur.com/C6Br0sh.png

Customizable Shift-Click broken on potions
holding shift: https://i.imgur.com/8EXb2zF.png
shift-click set to "use": https://i.imgur.com/hh6ks0t.png

**Comment by se7enAte9** [comment]
The quick-open issue is being caused by this `else if` in line [1047](https://github.com/runelite-extended/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java#L1047) of the MenuEntrySwapper class.

`else if (config.swapTravel() && (option.equals("pass") || option.equals("open")))
{
    swap(client, "pay-toll", option, target, false);
}`

**Comment by se7enAte9** [comment]
The holding shift issue is being caused by `event.getIdentifier()` being used instead of `event.getType()` in line [1213](https://github.com/runelite-extended/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java#L1213) of the MenuEntrySwapper class.

**Comment by se7enAte9** [comment]
Finally, the game objects issue is being caused by `event.getIdentifier()` being used instead of `event.getType()` in lines [721](https://github.com/runelite-extended/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java#L721), [728](https://github.com/runelite-extended/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java#L728), and [734](https://github.com/runelite-extended/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java#L734) of the MenuEntrySwapper class.

I imagine anything else relying on MenuEntryAdded's `event.getIdentifier` that has not been replaced with `event.getType()` would also be broken.

**Comment by Kyleeld** [comment]
some fixed in #831, shift click still roken.

**Comment by xKylee** [comment]
@SRLJustin is th
